tests/test_meta/test_grouped_model.py

```diff
@@ -103,5 +105,122 @@ def test_chickweigt_string_groups():
     y = df['weight']
 
     # This should NOT raise errors
-    GroupedEstimator(LinearRegression(), groups=['diet']).fit(X, y).predict(X)
-    GroupedEstimator(LinearRegression(), groups=1).fit(X_np, y).predict(X_np)
+    GroupedEstimator(LinearRegression(), groups=['diet'], shrinkage=None).fit(X, y).predict(X)
```

Comment:
since default value is `shrinkage=None` ... is there a reason to explicitly add it?

Reply:
No, removed it